aiqleads/__init__.py

```
@@ -1,3 +1,13 @@
\ No newline at end of file
+This package provides a lead marketplace platform with AI-driven recommendations.
+"""
+
+__version__ = "0.1.0"
+__author__ = "AIQLeads Team"
+__description__ = "AI-powered lead marketplace for real estate professionals"
+
+from .core import project_tracking
+from .utils import logging
+
+__all__ = ['project_tracking', 'logging']
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)